README.md

```diff
@@ -86,11 +86,11 @@ DVSA sends receipts in the email (which will help you in hacking it). You can us
 
 **_Note_**: each user will be assigned an email from `mailsac.com` which will be automatically verified. Real emails will be sent to their account and will appear in the application Inbox page. All this is **transparent** to the user and the deployer).
 
-**_Note_**: to make the email verification script work your dafault AWS region has to be "US East (N. Virginia)", for example by setting `region = us-east-1` in your ~/.aws/config file 
+**_Note_**: to make the email verification script work your default AWS region has to be "US East (N. Virginia)", for example by setting `region = us-east-1` in your ~/.aws/config file 
```

Comment:
This isn't a typo, but I tested the verification script with us-west-2 and it works too. SES is available in eu-west-1 as well.